libs/application/graphql/src/lib/mutations/uploadSignedFile.ts

```diff
@@ -0,0 +1,9 @@
+import { gql } from '@apollo/client'
+
+export const UPLOAD_SIGNED_FILE = gql`
+  mutation UploadSignedFile($input: UploadSignedFileInput!) {
+    uploadSignedFile(input: $input) {
+      externalData
```

Comment:
Wondering about this flow. 
Maybe we shouldn't request anything since we are not adding anything to the application just waiting and seeing if the request fails or succeeds. 
What do you think?

Reply:
I'm not sure how to go about doing this, I would like to investigate at a later date 👍